crates/compiler/test_gen/src/helpers/wasm.rs

```diff
@@ -400,7 +402,7 @@ macro_rules! assert_refcounts {
         let result =
             $crate::helpers::wasm::assert_wasm_refcounts_help::<$ty>($src, phantom, num_refcounts);
         match result {
-            Err(msg) => panic!("{:?}", msg),
```

Comment:
Same comment as above

Reply:
fixed in https://github.com/roc-lang/roc/compare/100a2df04327f19680e578c0f1bb2b87355b4311..e5eb5717eace1559d73561099e9381630b497634